pages/content/amp-dev/documentation/guides-and-tutorials/index.html

```diff
@@ -1,5 +1,5 @@
 ---
-$title: Learn AMP with Guides and tutorials
+$title: Guides and tutorials
```

Comment:
`$title` is also the one used for the `<title>` element and is therefore SEO relevant and should be more explaining than just "Guides & Tutorials". What was the reasoning you changed it?

Reply:
Longo asked me to change it